dockstore-client/src/main/java/io/dockstore/common/FileProvisionUtil.java

```diff
@@ -68,28 +74,49 @@ private FileProvisionUtil() {
         // disable utility constructor
     }
 
+    /** An extension of the Http4sFileProvider class to allow for setting the cookie specification to something
+     * other than DEFAULT. This cannot be set through the fileSystemOptions argument.
+     * Github responds with a cookie header containing a date in 4-digit year format.
+     * Cookie spec DEFAULT only allows 2-digit years; STANDARD allows 4-digits.
+     * Addresses #2261 warning messages, see https://github.com/dockstore/dockstore/issues/2261
+     */
+    static private class CustomHttp4sFileProvider extends Http4sFileProvider {
```

Comment:
I know I gave you contradictory feedback on this. I think it's OK like this, but you can also move it back to the way it was.

Reply:
Okay. I moved it back just because